File: C#/Easy/Concatenation-of-Array.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int[] ans = new int[nums.Length+nums.Length];
        for (int i = 0; i < nums.Length; i++)
        {
            ans[i] = nums[i];
            ans[nums.Length+i] = nums[i]; 
        }
        return ans;
    }
}

File: JavaScript/Easy/Concatenation-of-Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
const getConcatenation = function (nums) {
  const ans = []
  for (let i = 0; i < nums.length; i++) {
    ans[i] = nums[i]
    ans[nums.length + i] = nums[i]
  }
  return ans
}
